UnityProject/Assets/Scripts/Food/DrinkableContainer.cs

```diff
@@ -71,11 +72,31 @@ private void Drink(PlayerScript eater, PlayerScript feeder)
 		var drinkAmount = container.TransferAmount;
 		container.TakeReagents(drinkAmount);
 
+		DoDrinkEffects(eater, drinkAmount);
+
 		// Play sound
 		if (item && !string.IsNullOrEmpty(sound))
 		{
 			SoundManager.PlayNetworkedAtPos(sound, eater.WorldPos, sourceObj: eater.gameObject);
 		}
+	}
+
+	private void DoDrinkEffects(PlayerScript eater, float drinkAmount)
+	{
+		var playerEatDrinkEffects = eater.GetComponent<PlayerEatDrinkEffects>();
+
+		if(playerEatDrinkEffects == null) return;
 
+		if ((int) drinkAmount == 0) return;
+
+		foreach (var reagent in container.CurrentReagentMixGet)
+		{
+			//if its not alcoholic skip
+			if (!AlcoholicDrinksSOScript.Instance.AlcoholicReagents.Contains(reagent.Key)) continue;
+
+			//The more different types of alcohol in a drink the longer you get drunk for each sip.
+			playerEatDrinkEffects.ServerSendMessageToClient(eater.gameObject, (int)drinkAmount);
+			playerEatDrinkEffects.ServerSendMessageToClient(eater.gameObject, 0);
```

Comment:
why does it send it then send 0?

Reply:
Ah, good catch, thats was when i was testing with sync vars.